app/styles/components.less

```diff
@@ -0,0 +1,79 @@
+@info-row-logo-side-size: @info-row-height / 1.5;
+@info-box-width: 110px;
+@expand-button-width: 25px;
+.info-row {
+  position: relative;
+  list-style-type: none;
+  height: @info-row-height;
+
+  &:before {
+    content: '';
+    position: absolute;
```

Comment:
I think we could make mixing out of this vertical align combo. Would make the code cleaner and more readable.